controllers/storagecluster/cephobjectstoreusers.go

```diff
@@ -29,6 +31,19 @@ func (r *StorageClusterReconciler) newCephObjectStoreUserInstances(initData *ocs
 				Store:       generateNameForCephObjectStore(initData),
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      prometheusUserName,
+				Namespace: initData.Namespace,
+			},
+			Spec: cephv1.ObjectStoreUserSpec{
+				DisplayName: prometheusUserName,
+				Store:       generateNameForCephObjectStore(initData),
+				Capabilities: &cephv1.ObjectUserCapSpec{
+					User: "*",
```

Comment:
Why do we need a wildcard? Why `User`? It will be good to understand a bit more which permissions are required.

Reply:
Read permission is fine, the the quota is set on the internal user of OBC, so we need to fetch the user stats